smithy-model/src/main/java/software/amazon/smithy/model/loader/SetResourceBasedTargets.java

```diff
@@ -71,6 +71,8 @@ public void modifyMember(
         if (resource.getIdentifiers().containsKey(name)) {
             memberBuilder.target(resource.getIdentifiers().get(name));
         }
-        // TODO: check for resource properties too once they're added
+        if (resource.getProperties().containsKey(name)) {
```

Comment:
Do we have tests that cover resource property target elision (positive and negative)?

Reply:
Added a negative test and coverage in a positive test.